### PR TITLE
Remove timeout link from saga view when message not processed

### LINKED ISSUE #651: Saga view shows timeout hyperlink when message has not been processed
A user in a google issue asked about the display of timeouts in a saga view. Specifically when a timeout does not occur there is still a hyperlink that leads nowhere, which was confusing the user.

We should probably change the hyperlink to standard text if the timeout did not fire, and optionally also add a tooltip saying that.

cc @sergioc

### PR DESCRIPTION
**Pull request by adamralph**
fixes #651

### CODE REVIEW

**Comment by distantcam** [comment]
We've decided to go with this solution for now and improve the scrolling UI in another PR.

File: src/ServiceInsight/Saga/SagaUpdateControl.xaml [hunk 1]
```diff
@@ -509,21 +509,30 @@
...
-                                            <InlineUIContainer>
-                                                <TextBlock Text="Timeout Requested = " />
-                                            </InlineUIContainer><Run Text=" " />
-                                            <InlineUIContainer>
-                                                <TextBlock Text="{Binding TimeoutFriendly}" />
-                                            </InlineUIContainer>
-                                        </Hyperlink>
-                                    </TextBlock>
-                                </StackPanel>
+                                <!--<StackPanel Orientation="Horizontal">-->
```
Comment (distantcam) [inline src/ServiceInsight/Saga/SagaUpdateControl.xaml]: Should remove this comment

File: src/ServiceInsight/Saga/SagaUpdateControl.xaml [hunk 2]
```diff
@@ -509,21 +509,30 @@
...
+                                        FontSize="16px"
+                                        FontWeight="Bold">
+                                    <InlineUIContainer>
+                                        <TextBlock Text="Timeout Requested = " />
+                                    </InlineUIContainer>
+                                    <InlineUIContainer>
+                                        <TextBlock Text="{Binding TimeoutFriendly}" />
+                                    </InlineUIContainer>
+                                </TextBlock>
+                                <!--</StackPanel>-->
```
Comment (distantcam) [inline src/ServiceInsight/Saga/SagaUpdateControl.xaml]: ... and this one

**Comment by adamralph** [comment]
I amended the commit to remove the commented out `StackPanel` tags. This should be good to go now.

**Comment by adamralph** [comment]
Also rebased onto master, to keep things a little tidier.

**Pull request merged**